go.sum

```diff
@@ -37,14 +37,23 @@ github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZb
 github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
 github.com/riverqueue/river v0.7.0 h1:STWnPn0APPKQkAjsJG5Q8NzFOxPXI/BKNAL6dPT7oI8=
 github.com/riverqueue/river v0.7.0/go.mod h1:ulIQc2U1mUTQNTnInytnVu4vhU9GJGeWReXtndvLBXI=
+github.com/riverqueue/river v0.8.0 h1:IBUIP9eZX/dkLQ3T+XNNk0Zi7iyUksZd4aHxQIFChOQ=
+github.com/riverqueue/river v0.8.0/go.mod h1:EHRbhqVXDpXQizFh4lndwswu53N0txITrLM2y3vOIF4=
```

Comment:
Is this missing a `go mod tidy` or are we using both versions somehow?

Reply:
Ah I just ran `go mod tidy` and it cleaned a few of these out.